pytorch_lightning/utilities/memory.py

```diff
@@ -99,7 +99,7 @@ def garbage_collection_cuda() -> None:
             raise
 
 
-def get_memory_profile(mode: str) -> Union[Dict[str, float], Dict[int, float]]:
+def get_memory_profile(mode: str) -> Union[Dict[str, float]]:
```

Comment:
```suggestion
def get_memory_profile(mode: str) -> Dict[str, float]:
```
was this the intention?

Reply:
Yep, thanks!